src/comp/comp.py

```diff
@@ -24,48 +28,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human.name for human in humans if human.name[len(human.name) - 1] == "e"]
```

Comment:
Don't let JavaScript ruin your life! Come to the Python side: 
`b = [human.name for human in humans if human.name[-1] == "e"]`